fuzz/src/test/java/tech/pegasys/teku/fuzz/FuzzUtilTest.java

```diff
@@ -61,6 +62,7 @@
 import tech.pegasys.teku.spec.schemas.SchemaDefinitionsCapella;
 
 @ExtendWith(BouncyCastleExtension.class)
+@Disabled
```

Comment:
I think I need to regenerate files, haven't looked closely at this yet.

Reply:
@jtraglia is it hard to regenerate the pyspec tests? I'm not sure where I'd even need to start... It looks like you're the last person in this space...

Reply:
I think you should be able to use the generated tests in this repo:

* https://github.com/ethereum/consensus-spec-tests

Reply:
The generators doco didn't really cover having random files like we do, at least as far as I could understand...
I'll go with copying, should be able to find reference data for each case from that repo - thanks...